extensions/cocostudio/action/CCActionNode.js

```diff
@@ -86,7 +87,19 @@ ccs.ActionNode = ccs.Class.extend(/** @lends ccs.ActionNode# */{
                 actionFrame.frameIndex = frameIndex;
                 actionFrame.setEasingType(frameTweenType);
                 actionFrame.setEasingParameter(frameTweenParameter);
-                actionFrame.setPosition(positionX, positionY);
+
```

Comment:
Could you please give us the reason about these revises. 
Thanks very much.
David